aspnetcore/security/data-protection/consumer-apis/purpose-strings-multitenancy.md

```diff
@@ -16,7 +16,7 @@ uid: security/data-protection/consumer-apis/purpose-strings-multitenancy
 
 Since an IDataProtector is also implicitly an IDataProtectionProvider, purposes can be chained together. In this sense provider.CreateProtector([ "purpose1", "purpose2" ]) is equivalent to provider.CreateProtector("purpose1").CreateProtector("purpose2").
 
-This allows for some interesting hierarchical relationships through the data protection system. In the earlier example of [Contoso.Messaging.SecureMessage](purpose-strings.md#data-protection-contoso-purpose), the SecureMessage component can call provider.CreateProtector("Contoso.Messaging.SecureMessage") once upfront and cache the result into a private _myProvider field. Future protectors can then be created via calls to _myProvider.CreateProtector("User: username"), and these protectors would be used for securing the individual messages.
```

Comment:
Please code fence _myProvider and _myProvider.CreateProtector("User: username") instead. This makes things consistent with other docs.